src/client/assets/styles/components/_command.scss

```diff
@@ -31,6 +31,10 @@ command {
             }
         }
     }
+
+    &.view-mode {
+        pointer-events: none;
```

Comment:
Make sure that all the event handlers for this button are disabled as well when in view mode. When we enable keyboard accessibility we don't want to be able to invoke the action through keyboard instead of mouse.